application/config/config.go

```diff
@@ -346,8 +346,17 @@ func (c *Config) LogPath() string {
 	defer c.m.Unlock()
 	return c.logPath
 }
-func (c *Config) SnykApi() string                        { return c.snykApiUrl }
-func (c *Config) SnykCodeApi() string                    { return c.snykCodeApiUrl }
+func (c *Config) SnykApi() string     { return c.snykApiUrl }
+func (c *Config) SnykCodeApi() string { return c.snykCodeApiUrl }
+func (c *Config) SnykUi() string {
+	parsedUrl, err := url.Parse(c.snykApiUrl)
```

Comment:
Be careful here... an empty string would not error here, but it'd have an invalid host.

[parsedUrl.IsAbs()](https://pkg.go.dev/net/url#URL.IsAbs) could help guard against that.

See https://go.dev/play/p/fO4rfoYnZsX

Reply:
Thanks Casey, I added the check for IsAbs.